### PR TITLE
Bashing people in the head has a chance to deconvert them during One Mind

### PR DESCRIPTION
**Pull request by Kierany9**
## About The Pull Request

see title and/or changelog, the biggest nerf to endgame hiveminds
(alt title: try hitting it really hard)

## Why It's Good For The Game

ngl, i just read this suggested somewhere and found it funny

## Changelog
:cl:
add: You can now bash awoken vessels over the head to deconvert them, just like revolutionaries
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/human/species.dm [hunk 1]
```diff
@@ -1455,6 +1455,10 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 						var/datum/antagonist/rev/rev = H.mind.has_antag_datum(/datum/antagonist/rev)
 						if(rev)
 							rev.remove_revolutionary(FALSE, user)
+						var/datum/antagonist/hivevessel/woke = H.is_wokevessel()
+						if(woke && woke.one_mind)
+							woke.one_mind.remove_member(target.mind)
```
Comment (nemvar) [inline code/modules/mob/living/carbon/human/species.dm]: Unneeded. on_removal already removes vessels from the onemind.

File: code/modules/mob/living/carbon/human/species.dm [hunk 2]
```diff
@@ -1455,6 +1455,10 @@ GLOBAL_LIST_EMPTY(roundstart_races)
...
+							target.mind.remove_antag_datum(/datum/antagonist/hivevessel)
```
Comment (nemvar) [inline code/modules/mob/living/carbon/human/species.dm]:
``` suggestion
							H.mind.remove_antag_datum(/datum/antagonist/hivevessel)

Comment (Kierany9) [reply code/modules/mob/living/carbon/human/species.dm]: i am genuinely ashamed of screwing up a four line change

File: code/modules/mob/living/carbon/human/species.dm [hunk 3]
```diff
@@ -1455,6 +1455,9 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 						var/datum/antagonist/rev/rev = H.mind.has_antag_datum(/datum/antagonist/rev)
 						if(rev)
 							rev.remove_revolutionary(FALSE, user)
+						var/datum/antagonist/hivevessel/woke = H.is_wokevessel()
+						if(woke)
```
Comment (nemvar) [inline code/modules/mob/living/carbon/human/species.dm]: Why did you remove the second conditional? That one was actually needed. Now it can unconvert brainwashed people before one mind even happens.

Comment (Kierany9) [reply code/modules/mob/living/carbon/human/species.dm]: intentional, at least it was half an hour ago
fixin' aaaa

**Comment by HomocidlePancakes** [comment]
When in doubt, apply toolbox to the forehead.

**Comment by OverlyGeneric** [comment]
oh boy, i can't wait to be harmbatoned by security for being one-minded without knowing.

**Comment by CPTANT** [comment]
just remove the gamemode

**Comment by anconfuzedrock** [comment]
great way to fix the symptom, not the disease.

**Comment by datorangebottle** [comment]
What CPTANT said.

But also, way to punish hiveminds for actually winning by making deconversion easier.

**Comment by Shadowflame909** [comment]
Is it actually an "I win" if the I win can be negated?

We should just make ascended hiveminds be able to gib people. Shadowlings style.

**Comment by wesoda25** [comment]
In its current state ascension isn’t a super difficult task, I’ve seen it done under 20 minutes. “Endgame” for any gamemode is lame, for example the time between nar sie summon and rouns end. Ascension shouldn’t be some big affair where the hivemind becomes invincible and the round is basically over. Giving the crew a chance to fight back is nice and I like it.

**Comment by Shadowflame909** [comment]
Well I completely disagree. Having a big grand game-changer as One Mind makes itself out to be on the wiki being throttled makes this feature less encouraged. Considering it instantly ends when the hivemind dies, and having my purple man army taken away from me and making me an obvious valid really doesn't fit hivemind at all. Considering the therefore lack of aggressive tactics they get.

All I'm saying is, may want to give hivemind glowy fireballs when they one mind. Otherwise that big draw of one mind will be pretty much negated.

**Comment by Kierany9** [comment]
>Is it actually an "I win" if the I win can be negated?
>Well I completely disagree. Having a big grand game-changer as One Mind makes itself out to be on the wiki being throttled makes this feature less encouraged

One Mind is an endgame but not a game ender if that makes sense, the wiki even emphasizes this and tell the hosts to simply 'have fun' with their newfound command. There's a reason why it doesn't auto-call the shuttle like slings do or just flat out end the round like cult.

**Comment by anconfuzedrock** [comment]
one out of the 20 problems with hivemind is that there's very little sec can do about them going onemind short of implanting the entire crew (they can't hunt down and kill the hiveminds- by the time they realise the issue, and even if they successfully hunt down hiveminds, when they kill the second-to-last hivemind the final one will certainly onemind.) the problems with the gamemode aren't going to be fixed by making the catastrophe easier to deal with, the catastrophe shouldn't be happening in the first place. hivemind doesn't have any redeeming qualities and basically no completely unique assets or features besides standing-still conversion, and none of this is solving that issue.

**Comment by wesoda25** [comment]
can we please merge this already?
